.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: [main, polaris-release]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: 'Build, Lint, and Test: node-${{ matrix.node_version }}'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: ['14.17.0', '16.13.0']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Setup Node to ${{ matrix.node_version }}
+        uses: actions/setup-node@v3
+        with:
+          node_version: ${{ matrix.node_version }}
+          cache: yarn
+
+      - name: Restore cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/*.tsbuildinfo
+            **/.turbo
+            node_modules/.cache/turbo
```

Comment:
I'm noticing some of the removed workflows cached `.eslintcache`. Should we add that here?

Reply:
Additionally, we likely need to port the `polaris-react/.loom` cache (removed from the `ci-polaris-react` workflow)

Reply:
Added restoration of test and lint cache 🔧